CONTRIBUTING.md

```diff
@@ -14,26 +14,27 @@ Found a security issue? Please disclose it responsibly. We have a team of develo
 
 ## Contributing to translations
 
-We use [WebTranslateIt][webtranslateit] to manage translation. Please read [our wiki page][translation-wiki] to know more about it. Except if your language isn't on webtranslateit, please do **not** open a pull request to update translations.
+We use [WebTranslateIt][webtranslateit] to manage translations of the app interface. Please read [our wiki page][translation-wiki] to find out more about this. If your language is on WebTranslateIt, please do **not** open a pull request to update translations. If you want to contribute to a language that is not featured on WebTranslateIt, please contact our developers to find out how to proceed.
```

Comment:
I added that because I \*think\* that is how things have been done in the past for a new language. But I'm by no means sure. I suppose it would have been better if I'd have asked Jonne and @denschub rather than just adding that sentence to my PR! Hopefully they can confirm for you what is the proper procedure.